src/FakeClient/FakeClientMain.cpp

```diff
@@ -293,63 +294,64 @@ int main(int argc, char* argv[]) {
           (absl::GetFlag(FLAGS_instrument_offset) == 0),
       "Binary path and offset of the function to instrument need to be specified together");
 
-  uint32_t process_id = absl::GetFlag(FLAGS_pid);
-  if (process_id == 0) {
+  CaptureClientOptions options;
+  options.process_id = absl::GetFlag(FLAGS_pid);
+  if (options.process_id == 0) {
     const std::string pid_file_path = absl::GetFlag(FLAGS_pid_file_path);
     ORBIT_FAIL_IF(pid_file_path.empty(), "A PID or a path to a file is needed.");
     WaitForFileModification(pid_file_path);
-    process_id = ReadPidFromFile(pid_file_path);
+    options.process_id = ReadPidFromFile(pid_file_path);
   }
-  ORBIT_LOG("process_id=%d", process_id);
-  ORBIT_FAIL_IF(process_id == 0, "PID to capture not specified");
+  ORBIT_LOG("process_id=%d", options.process_id);
+  ORBIT_FAIL_IF(options.process_id == 0, "PID to capture not specified");
 
-  uint16_t samples_per_second = absl::GetFlag(FLAGS_sampling_rate);
-  ORBIT_LOG("samples_per_second=%u", samples_per_second);
-  constexpr uint16_t kStackDumpSize = 65000;
-  const UnwindingMethod unwinding_method =
+  options.samples_per_second = absl::GetFlag(FLAGS_sampling_rate);
+  ORBIT_LOG("samples_per_second=%f", options.samples_per_second);
```

Comment:
```suggestion
  ORBIT_LOG("samples_per_second=%.0f", options.samples_per_second);
```
The flag is integer anyway.

Reply:
Done, thanks.